main/src/main/java/org/mobilitydata/gtfsvalidator/validator/AgencyConsistencyValidator.java

```diff
@@ -42,8 +42,12 @@
  */
 @GtfsValidator
 public class AgencyConsistencyValidator extends FileValidator {
+  @Inject
```

Comment:
```suggestion
  private final GtfsAgencyTableContainer agencyTable;
  
  @Inject
```

Reply:
I would declare agencyTable before injecting the constructor to keep consistency with the other validators.

Reply:
Agree, done.